src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1030,18 +1203,72 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
       if( fv3sar_bg_opt == 0) then 
          call gsi_fv3ncdf_read(grd_fv3lam_dynvar_ionouv,gsibundle_fv3lam_dynvar_nouv,fv3filenamegin%dynvars,fv3filenamegin)
          call gsi_fv3ncdf_read(grd_fv3lam_tracer_ionouv,gsibundle_fv3lam_tracer_nouv,fv3filenamegin%tracers,fv3filenamegin)
+        if (laeroana_fv3cmaq) then
+         call gsi_fv3ncdf_read(grd_fv3lam_tracerchem_ionouv,gsibundle_fv3lam_tracerchem_nouv,fv3filenamegin%tracers,fv3filenamegin)
+        endif
       else
          call gsi_fv3ncdf_read_v1(grd_fv3lam_dynvar_ionouv,gsibundle_fv3lam_dynvar_nouv,fv3filenamegin%dynvars,fv3filenamegin)
          call gsi_fv3ncdf_read_v1(grd_fv3lam_tracer_ionouv,gsibundle_fv3lam_tracer_nouv,fv3filenamegin%tracers,fv3filenamegin)
+       if (laeroana_fv3cmaq) then
+         call gsi_fv3ncdf_read_v1(grd_fv3lam_tracerchem_ionouv,gsibundle_fv3lam_tracerchem_nouv,fv3filenamegin%tracers,fv3filenamegin)
+       endif
       endif
 
-      if( fv3sar_bg_opt == 0) then 
-        call GSI_BundleGetPointer ( gsibundle_fv3lam_dynvar_nouv, 'delp'  ,ges_delp ,istatus );ier=ier+istatus
+     if (laeroana_fv3cmaq) then
+        ier=0
+        call gsi_bundlegetpointer_fv3lam_tracerchem_nouv
+        if (ier/=0) call die(trim(myname),'cannot get pointers for fv3lam_tracerchem_nouv, ier =',ier)
+
+! Compute background amassi, amassj, amassk and pm2_5   
+       do k=1,nsig
```

Comment:
Please align line 1223 with line 1220 and ensure at least two-space indentation for all do loops and if statements.